src/main.ts

```diff
@@ -41,3 +73,12 @@ export async function run(): Promise<void> {
     if (error instanceof Error) core.setFailed(error.message)
   }
 }
+
+export async function cleanup(): Promise<void> {
+  core.info('Removing docker context ...')
+  try {
+    execSync('docker context remove --force target', { stdio: [] })
+  } catch {
+    // Ignore
```

Comment:
I think we could still log something. I agree that `setFailed` makes little sense here, but maybe just `core.info('Docker context removal failed ...')` or `core.warn`, since we throw away the output of the command. What do you think?

Reply:
addressed with [6b64b60](https://github.com/simplificator/deploy-action/pull/2/commits/6b64b60a5070f86d51b9ecf821a19c533eb716f3).